.changeset/six-grapes-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend': patch
+---
+
+**BREAKING**: Update the router to use the new `auth` services. The router now requires a discovery service option to get credentials for the permission service.
```

Comment:
Can't have a breaking change unless we also make this a `major` bump :)

Similarly to the scaffolder/catalog I think we can default the `discovery` implementation to `HostDiscovery` internally, like this:

https://github.com/backstage/backstage/blob/15ba00ff7dc25828d797f2615b09e49c522d3090/plugins/catalog-backend/src/service/CatalogBuilder.ts#L460

Reply:
Updated